composer.json

```diff
@@ -1,37 +1,44 @@
 {
-    "name": "bluehost/maestro-connector",
-    "description": "Give trusted web professionals admin access to your WordPress account. Revoke anytime.",
-    "type": "wordpress-plugin",
-    "license": "GPL-2.0-or-later",
-    "authors": [
-        {
-            "name": "William Earnhardt",
-            "email": "wearnhardt@gmail.com"
-        }
-    ],
-    "repositories": [
-        {
-        "type": "composer",
-        "url": "https://satis.wpteamhub.com"
-        }
-    ],
+  "name": "bluehost/maestro-connector",
+  "description": "Give trusted web professionals admin access to your WordPress account. Revoke anytime.",
+  "type": "wordpress-plugin",
+  "license": "GPL-2.0-or-later",
+  "authors": [
+    {
+      "name": "William Earnhardt",
+      "email": "wearnhardt@gmail.com"
+    }
+  ],
+  "repositories": [
+    {
+      "type": "composer",
+      "url": "https://bluehost.github.io/satis/"
+    }
+  ],
+  "autoload": {
+    "psr-4": {
+      "Bluehost\\": "inc/"
+    }
+  },
+  "require": {
+    "firebase/php-jwt": "^5.0"
+  },
+  "require-dev": {
+    "bluehost/wp-php-standards": "^1.1"
+  },
+  "config": {
+    "optimize-autoloader": true
+  },
     "scripts": {
+        "fix": [
+            "vendor/bin/phpcbf . --standard=Bluehost"
+        ],
         "lint": [
-        "vendor/bin/phpcs ."
+            "vendor/bin/phpcs . --standard=Bluehost -s"
         ]
     },
-    "autoload": {
-        "psr-4": {
-        "Bluehost\\": "inc/"
-        }
-    },
-    "require": {
-        "firebase/php-jwt": "^5.0"
-    },
-    "require-dev": {
-        "bluehost/wp-php-standards": "^1.1"
-    },
-    "config": {
-        "optimize-autoloader": true
+    "scripts-descriptions": {
+        "fix": "Automatically fix coding standards issues where possible.",
+        "lint": "Check files against coding standards."
     }
 }
```

Comment:
Looks like you switched the `yml` file to two space indention and I had 4, but missed this section.

Reply:
I fixed this